src/pudl/metadata/resources/eia861.py

```diff
@@ -439,6 +555,14 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_operational_data_misc": {
+        # TODO: misc might be a missleading name
+        "description": (
+            """The annual megawatt hours (MWH) for all a utility's sources of
```

Comment:
We've got a couple of electricity "sources and dispositions" tables in the FERC Form 1. Would it be easy to name this table so that it's clearly analogous? Would be interesting to compare them:

- `core_ferc1__yearly_energy_sources_sched401`
- `core_ferc1__yearly_energy_dispositions_sched401`

Maybe:

- `core_eia861__yearly_energy_sources_dispositions`

It seems a little confusing to list the sources, but not the dispositions in the description.  I think it's fine to not enumerate them at all, since if a user knows it's a sources & dispositions table, the column names are readily interpretable, and there's kind of a lot of them.

Reply:
We could rename this table, but it would distance the table from the original EIA861 table name (`Operational Data`). Which is fine, but it make make it more confusing for folks that have previously worked with the raw data.

Are you referring to both of the `operational_data` tables here or just the `misc` one?

Reply:
To be addressed in #3301